configure

```diff
@@ -388,6 +388,7 @@ msg "inspecting environment"
 
 CFG_OSTYPE=$(uname -s)
 CFG_CPUTYPE=$(uname -m)
+ENDIAN=$(printf '\1' | od -dAn)
```

Comment:
Here `od` isn't `need_cmd`'d, thus unsafe to rely on. You could refer to the [`rustup-init.sh`] implementation for an alternative that only requires `head` and `tail`.

[`rustup-init.sh`]: https://github.com/rust-lang-nursery/rustup.rs/blob/1bedbb1b81a9a4cf6c8927db27a957e8b1fa18bb/rustup-init.sh#L125-L142